functions/menu.bash

```diff
@@ -154,6 +158,9 @@ show_main_menu() {
       43\ *) nginx_setup ;;
       *Delay\ rules\ load) delayed_rules yes;;
       *Default\ order) delayed_rules no;;
+      *Zulu\ OpenJDK\ 32-bit) update_config_java "Zulu32" && java_install_or_update "$java_opt";;
```

Comment:
what if java_opt=Zulu64 or Adopt ?

Reply:
What do you mean?

Reply:
java_install_or_update "$java_opt"

Reply:
Well it should not be possible for them to update as we are setting $java_opt to Zulu32 right before executing that.

Reply:
in update_config_java() ?
Nevertheless exchange the variable here for a fixed "Zulu32"